pkg/MlNetMklDeps/MlNetMklDeps.nupkgproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>Intel(R) MKL dependencies for ML.Net.</PackageDescription>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="../Microsoft.ML/Microsoft.ML.nupkgproj" />
```

Comment:
Can we write down how exactly it's gonna work?
How it looks in my head based on conversations (can be far away from reality)
1. Have a internal repo with pre build custom dlls for each OS.
2. Publish them to nuget.org
3. Reference them in Microsoft.ML nuget as dependency

this steps have nothing to do with this package file, so I obviously don't understand something.